app/interactors/outdated_prolem_clearer.rb

```diff
@@ -0,0 +1,32 @@
+require 'problem_destroy'
+
+class OutdatedProblemClearer
+
+  ##
+  # Clear all problem not present for more than one week.
+  #
+  def execute
+    nb_problem_outdated.tap { |nb|
+      if nb > 0
+        criteria.each do |problem|
+          ProblemDestroy.new(problem).execute
+        end
+        repair_database
+      end
+    }
+  end
```

Comment:
I'd like to add one test here just to execute at least one success case. You can use a mock to set the config temporarily since it won't have a default setting. Something like
```ruby
allow(Errbit::Config).to receive(:errbit_problem_destroy_after_days).and_return(30)
```